.githooks/commit-msg

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+## the first arg is the path to the git commit temporary file
+TEMPORARY_FILE_PATH=$1
```

Comment:
Don't have much knowledge on these hooks, will there be any scenario where the `TEMPORARY_FILE_PATH` might be undefined?

Reply:
According to their [documentation](https://git-scm.com/docs/githooks#_commit_msg), this hook is called with 1 parameter, that contains `the name of the file that holds the proposed commit log message`. I assume it will always be defined.